src/modules/service-catalog/components/service-catalog-list-item/LoadingState.tsx

```diff
@@ -0,0 +1,64 @@
+import { Col, Grid, Row } from "@zendeskgarden/react-grid";
+import { Skeleton } from "@zendeskgarden/react-loaders";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+import { MD, XXXL } from "@zendeskgarden/react-typography";
+import styled from "styled-components";
+
+const SkeletonItem = styled(Col)`
+  display: flex;
+  flex-direction: column;
+  gap: ${(props) => props.theme.space.sm};
+  height: 100%;
+  border-radius: ${(props) => props.theme.borderRadii.md};
+  padding: ${(props) => props.theme.space.md};
+  border: ${(props) => props.theme.borders.sm}
+    ${(props) => getColorV8("grey", 300, props.theme)};
+`;
+
+const TextSkeleton = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${(props) => props.theme.space.xxs};
+  margin-bottom: ${(props) => props.theme.space.md};
+`;
+
+const StyledGrid = styled(Grid)`
+  padding: 0;
+`;
+
+const DescriptionSkeleton = styled.div`
+  font-size: ${(props) => props.theme.space.xs};
+`;
+
+const SkeletonCol = () => (
+  <Col xs={12} sm={6} md={4} lg={3}>
+    <SkeletonItem>
+      <XXXL>
```

Comment:
Why wrap in `XXXL`? We can just set the skeleton height to 40px

Reply:
I thought about having less fixed values as it takes hight form line hight but I will change it back to 40px

Reply:
Yeah but in this case, it is a placeholder for the icon and there is no text or line-height, so we should set the height to a fixed value as written in the Garden doc:
> By default, a Skeleton loader’s height is 60% of the font size. To make placeholders for other components, such as buttons, height and width can be explicitly set.